src/main/java/io/github/syst3ms/skriptparser/expressions/ExecExprListOperators.java

```diff
@@ -166,7 +175,8 @@ public boolean init(Expression<?>[] expressions, int matchedPattern, ParseContex
 					up = temp + 1;
 				}
 				if (low == -1 || up == -1 || up == 0 || st == 0 || low > up) {
-					list.change(ctx, new Object[0], ChangeMode.SET);
+					if (isEffect)
+						list.change(ctx, new Object[0], ChangeMode.SET);
 					return new Object[0];
```

Comment:
why not adding a warning that can help skript-parser users why they list become empty list ? 
```suggestion
                                        new SkriptLogger().warn("Skript lists start at one, so you cannot extract negatives or zero values from a list")
					return new Object[0];
```

Reply:
We cannot add runtime warnings. The logger wasn't made for such tasks and it is not the best practice anyways.

When we introduce runtime-errors (not sure how yet), all these empty objects will get a meaning.